proto/xudrpc.proto

```diff
@@ -422,6 +426,8 @@ message OrderRemoval {
   string local_id = 4 [json_name = "local_id"];
   // Whether the order being removed is a local own order or a remote peer order.
   bool is_own_order = 5 [json_name = "is_own_order"];
+  // The removal time for the Order.
```

Comment:
"The epoch time in milliseconds when the order was removed from the order book."